Wireframe/index.html

```diff
@@ -3,11 +3,83 @@
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+
+      <!-- NAV -->
+       <nav>
+        <ul>
+          <l><a href="#what">What</a></li>
+            <l><a href="#what">Why</a></li>
+              <l><a href="#when">When</a></li>
+        </ul>
+       </nav>
+
+       <!-- header -->
+       <header>
+        <h1>Understanding Git</h1>
+        <p class="header-info">A beginner's guide to <em><strong><span class="text-border">VERSION CONTROL</span></strong></em><br> and branching in Git </p>
+        <button>more</button>
+       </header>
+
+      <!-- ARTICLES SECTION -->
+
+     <section class="main-section">
+
+       <article id="what">
+        <h2>What is Git?</h2>
+        <p> Git is a distributed version control system that allows <mark ><em> multiple developers</mark></em> to work on a project simultaneously.</p>
+        <a href="https://docs.github.com/en/get-started/start-your-journey/about-github-and-git" target="_blank">Learn more about Git</a>
+      </article>
+
+      <div id="what" class="cards-container">
+        <!-- Card 1 -->
+        <article class="card">
+          <h2 class="card-title">Why Do Developers Need Git?</h2>
+          <p class="card-info">Git is all about efficiency.Git helps developers track changes, collaborate on code, and manage versions of their projects effectively.</p>
+          <a href="https://medium.com/@iamsaura8h/why-github-is-essential-for-your-tech-career-f8c19e0a9100" target="_blank">See why Git is essential</a>
+        </article>
+          <!-- Card 2 -->
+      <article id="when" class="card">
+        <h2 class="card-title">When you need to create a branch?</h2>
+        <p class="card-info">You should create a branch when you want to add a new feature, fix a bug, or experiment with new ideas without affecting the main codebase</p>
+        <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches" target="_blank">Learn more about branches</a>
+      </article>
+      </div>
+      
+    </section>
+
+       <!-- Footer-->
+        <footer>
+          <div class="info">
+          <h2>SAY HELLO</h2>
+          <p class="p-footer">Della Bella</p>
+          <p class="p-footer">3 Moon Street, London</p>
+          <p>&copy; 2024 Git Learning</p>
+        </div>
+        <div class="social">
+          <ul>
+            <li> 
+              <a class="footer-links" href="https://www.dropbox.com/en_GB/" target="_blank" >
+                <img src="img/dropbox.png" alt="gmail logo" height="60px" width="auto">
+              </a>
+            </li>
+              <a class="footer-links" href="https://github.com/" target="_blank">
```

Comment:
I think that there are rules in html about parent and child elements. Can you look up the rules for ul / unordered list, and tell me what "child" elements ul can have?

This page might help https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Reply:
<ul> The main rule is that only <li> elements are allowed as children.
fixed.